src/components/views/rooms/RoomSublist.tsx

```diff
@@ -657,7 +657,6 @@ export default class RoomSublist extends React.Component<IProps, IState> {
 
                     const badge = (
                         <NotificationBadge
-                            forceCount={true}
```

Comment:
and here?

Reply:
This was forcing the badge to always have a count, or not be shown at all: since we now do want to show an empty badge, this isn't what ewe want anymore.

Reply:
mmmkay, But presumably this was doing something important before, like hiding the badge when there were no notifications? Do we really want to show an empty badge in that situation now?

Reply:
Okay, so I think this was actually a bug in that the room sublists did show a badge when they weren't supposed to. I've fixed this in ff7d281f54 and also renamed the option to, hopefully, make it betetr describe what it actually does. It's also tested now.